packages/metals-languageclient/src/fetchMetals.ts

```diff
@@ -17,15 +18,21 @@ interface PackedChildPromise {
 
 export async function fetchMetals({
   serverVersion,
-  javaConfig: { coursier },
+  serverProperties,
+  javaConfig: { javaOptions, coursier, extraEnv },
 }: FetchMetalsOptions): Promise<PackedChildPromise> {
   const serverDependency = calcServerDependency(serverVersion);
 
+  const fetchProperties = serverProperties.filter(
```

Comment:
I would just warn here, since we do debug metals sometimes and that requires setting debug settings.

Alternatively we could check if it's set to quiet=y.

Reply:
I restored that `filter` call [from v1.29.0](https://github.com/scalameta/metals-vscode/blob/v1.29.0/packages/metals-languageclient/src/fetchMetals.ts#L30), but I'm not familiar with why it was there to being with.

To make sure I understand, are you asking to emit a warning message if `-agentlib` is present indicating that it isn't being passed to Coursier?

Reply:
Ach right, my bad. I forgot it was just for coursier, I assumed too much :sweat: It's fine as it is, thanks!